src/components/Slider/Slider.js

```diff
@@ -94,7 +93,11 @@ export class RawSlider extends Component {
       pips === false ? [] : this.computeDefaultPitPoints({ min, max });
 
     return (
-      <div className={this.isDisabled ? 'ais-range-slider--disabled' : ''}>
+      <div
+        className={cx(suit(), {
```

Comment:
This doesn't allow to customize the root CSS class yet.

Reply:
Do you need some more time to implement this one?

Reply:
Yes, I plan to do it today.

I spotted this issue after requesting your review, sorry about that.

Reply:
No worries at all. It was on opportunity to take a look already :)